docs/seps/sep-0005.rst

```diff
@@ -1 +1,89 @@
+================================
+SEP 5 — SDyPy Unified timeseries
+================================
+
+:Authors: Wout Weijtjens <wout.weijtjens@vub.be>, 
```

Comment:
I suggest adding all people who have/will contribute.

Reply:
Of course :-)